PZNS_Framework/media/lua/client/05_npc_actions/PZNS_RunTo.lua

```diff
@@ -141,6 +141,8 @@ function PZNS_RunToSquareXYZ(npcSurvivor, squareX, squareY, squareZ)
         squareZ  -- Floor level
     );
 
-    local runAction = PZNS_RunToTimedAction:new(npcIsoPlayer, targetSquare);
-    PZNS_UtilsNPCs.PZNS_AddNPCActionToQueue(npcSurvivor, runAction);
+	if targetSquare ~= nil then
```

Comment:
OK, this makes sense - once the square is unloaded from game-memory, it becomes a **nil** value.